pycparser/_ast_gen.py

```diff
@@ -134,6 +134,22 @@ def _gen_children(self):
 
         return src
 
+    def _gen_eq(self):
+        src  = '    def __eq__(self, o):\n'
+        src += '        if not o: return False\n'
```

Comment:
Is this comparing children?

Reply:
Which particular case do you mean?

Reply:
`if self.{0} != o.{0}` will trigger recursive `__ne__` → `__eq__` calls. If you mean inner members.